core/templates/services/html-length.service.spec.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for HtmlLengthService.
+ */
+
+import { TestBed } from '@angular/core/testing';
+import { DomSanitizer} from '@angular/platform-browser';
+import { SecurityContext } from '@angular/core';
+import { HtmlLengthService } from 'services/html-length.service';
+import { LoggerService } from './contextual/logger.service';
+
+class MockLoggerService {
+  error(message: string) {}
+}
+
+class MockDomSanitizer {
+  sanitize(context: SecurityContext, value: string): string {
+    return value;
+  }
+}
+
+describe('Html Length Service', () => {
```

Comment:
Can you add a test case that covers `ol` and `ul`?

Reply:
@Ash-2k3 I don't see the new test cases that cover `ol` and `ul`. Can you please point me to them?